internal/storage/types.go

```diff
@@ -30,6 +30,12 @@ type AddressStats struct {
 	mutex   sync.RWMutex `rlp:"-"`
 }
 
+type Account struct {
+	Address string       `json:"address"`
+	Balance *big.Int     `json:"balance"`
+	Mutex   sync.RWMutex `rlp:"-"`
```

Comment:
If I am not mistaken you aren't changing this concurrently, so you don't need mutex here